test/spspSpec.js

```diff
@@ -96,170 +93,78 @@ describe('SPSP Module', function () {
-      yield expect(SPSP.quoteSource(undefined, 'alice@example.com', '10')).to.eventually.be.rejected
-    })
-
-    it('should fail without an identifier', function * () {
-      yield expect(SPSP.quoteSource(this.plugin, undefined, '10')).to.eventually.be.rejected
-    })
-
-    it('should fail without an amount', function * () {
-      yield expect(SPSP.quoteSource(this.plugin, 'alice@example.com')).to.eventually.be.rejected
-    })
-  })
-
-  describe('sendPayment', function () {
-    beforeEach(function * () {
-      nock('https://example.com')
-        .get('/.well-known/webfinger?resource=acct:alice@example.com')
-        .reply(200, webfinger)
-
-      nock('https://example.com')
-        .get('/spsp')
-        .reply(200, spspResponse)
-
-      this.payment = yield SPSP.quoteDestination(this.plugin, 'alice@example.com', '10')
-    })
 
-    it('should successfuly send a payment', function * () {
-      let payment
-      const client = new MockClient({})
-
-      client.sendQuotedPayment = (paymentParams) => (new Promise((resolve) => {
-        payment = paymentParams
-        setImmediate(() => client.emit('outgoing_fulfill', {
-          executionCondition: paymentParams.executionCondition
-        }, 'fulfillment'))
-        resolve()
-      }))
-
-      const result = yield SPSP.sendPayment(this.plugin, this.payment, { client })
-      expect(result).to.deep.equal({ fulfillment: 'fulfillment' })
-
-      expect(payment).to.be.ok
-      expect(payment.destinationAmount).to.equal(this.payment.destinationAmount)
-      expect(payment.sourceAmount).to.equal(this.payment.sourceAmount)
-      expect(payment.uuid).to.equal(this.payment.id)
-      // payment destinationAccount will have extra PSK identifiers
-      expect(payment.destinationAccount.startsWith(this.payment.destinationAccount)).to.be.ok
-    })
-
-    it('should fail without plugin', function * () {
-      yield expect(SPSP.sendPayment(undefined, this.payment))
-        .to.eventually.be.rejectedWith(/missing plugin/)
-    })
-
-    it('should fail without payment', function * () {
-      yield expect(SPSP.sendPayment(this.plugin, undefined))
-        .to.eventually.be.rejectedWith(/missing payment/)
-    })
-
-    it('should fail without destinationAccount', function * () {
-      delete this.payment.destinationAccount
-      yield expect(SPSP.sendPayment(this.plugin, this.payment))
-        .to.eventually.be.rejectedWith(/missing destinationAccount/)
-    })
-
-    it('should fail without destinationAmount', function * () {
-      delete this.payment.destinationAmount
-      yield expect(SPSP.sendPayment(this.plugin, this.payment))
-        .to.eventually.be.rejectedWith(/missing destinationAmount/)
-    })
-
-    it('should fail without sourceAmount', function * () {
-      delete this.payment.sourceAmount
-      yield expect(SPSP.sendPayment(this.plugin, this.payment))
-        .to.eventually.be.rejectedWith(/missing sourceAmount/)
-    })
```

Comment:
https://github.com/interledgerjs/ilp/pull/79/files#diff-b557f2459a4c96ddfe2567537174fa7aR100 it's just checking all these asserts to make sure the arguments are there.